ctd/read.py

```diff
@@ -380,8 +380,11 @@ def from_cnv(fname):
     )
     f.close()
 
+
     key_set = False
-    prkeys = ["prDM", "prdM", "pr"]
+    prkeys = [
+      "prM ", "prE", "prDM", "pr50M", "pr50M1", "prSM", "prdM", "pr"
```

Comment:
Can you add a comment on the top mentioned that these are the valid names for the seabird CNV format. If you have a link for that info please add that too.